### PR TITLE
Dismemberment tweak

### PR DESCRIPTION
**Pull request by yackemflam**
:cl: Yackemflam
tweak: Dismemberment chance on weapons are now a var. Admins can now fuck with shit to make things have 100% chance to dismember you, or 0% chance to dismember you.
/:cl:

### CODE REVIEW

**Comment by MMMiracles** [comment]
do you actually do all your edits with the online editor or

**Comment by Kromgar** [comment]
I'm not even a coder but every time you do this damn pr you get told to not do it in the online editor

**Comment by KorPhaeron** [comment]
It's a yackemflam mixing balance changes with a refactor episode

File: code/game/gamemodes/changeling/powers/mutations.dm [hunk 1]
```diff
@@ -153,6 +153,7 @@
 	flags = ABSTRACT | NODROP | DROPDEL
 	w_class = 5.0
 	force = 25
+	dismemberment_chance = 100
```
Comment (KorPhaeron) [inline code/game/gamemodes/changeling/powers/mutations.dm]: So it kills you in one hit?

**Comment by Aloraydrel** [comment]
Change the channel please

**Comment by yackemflam** [comment]
Kor, everything is exactly the same.
You can check yourself.

**Comment by KorPhaeron** [comment]
guess I'm just dumb

**Comment by yackemflam** [comment]
Why the thumbsdown?

**Comment by KorPhaeron** [comment]
Probably because you added another var to every item which is going to be the source of lots of bikeshedding in the future

**Comment by KorPhaeron** [comment]
Also this is pretty much entirely uneeded because you can already override 

/obj/item/proc/get_dismemberment_chance

And have it return 0 or 100 or whatever

**Comment by yackemflam** [comment]
But you also can have custom dismemberment chance permanently for a custom item. Because basing dismemberment off of force was fine, but w_class was a retarded idea.

**Comment by KorPhaeron** [comment]
If by custom you mean admin editable, then yes, this works for that better than procs.

However if admins really want to make an instant dismember banana peel they can already just give it 100000 armour penetration or something

**Comment by Bawhoppen** [comment]
*sigh*

yackem I was on your side before now even I am getting tired of your antics

**Comment by KorPhaeron** [comment]
@optimumtact 

What's your take on this?

Add a var or have coders override procs?

**Comment by PKPenguin321** [comment]
This is dumb and stupid

**Comment by optimumtact** [comment]
all weapon properties should be moved to a weapon datum that can be assigned to each atom as needed.

Looking forward to this refactor.

**Pull request closed without merging**